application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadCreatedListener.java

```diff
@@ -41,7 +39,7 @@ public HelpThreadCreatedListener(HelpSystemHelper helper) {
     }
 
     @Override
-    public void onChannelCreate(@Nonnull ChannelCreateEvent createEvent) {
+    public void onChannelCreate(ChannelCreateEvent createEvent) {
```

Comment:
can u double check other `EventReceiver`s in the code base? Im pretty sure I copied this method from somewhere in the code base. So maybe that other user also has `@Nonnull`.

Reply:
ohh yea there was more